code/datums/browser.dm

```diff
@@ -109,94 +119,47 @@
 	if(written_text)
 		. = user.handle_reading_literacy(user, .)
 
-/datum/browser/proc/open(var/use_onclose = 1)
```

Comment:
You've searched for all instances of `open` and `update` and made sure they aren't using the old named args? People love using named args for these kinds of things.